Makefile

```diff
@@ -0,0 +1,8 @@
+test:
+	bats **/**/*.bats
```

Comment:
What's the **/** syntax?

Reply:
this is just file globbing `**` is to look into any directory